client/apps/Planning/PlanningList.tsx

```diff
@@ -206,6 +210,7 @@ export class PlanningListComponent extends React.PureComponent<IProps> {
                     listViewType={listViewType}
                     sortField={sortField}
                     indexItems
+                    searchParams= {currentSearch.advancedSearch}
```

Comment:
space after `=` not desirable for code style

Reply:
my bad !, I forgot to remove it